main.py

```diff
@@ -0,0 +1,51 @@
+import sys
+from PIL import Image
+from pathlib import Path
```

Comment:
didn't say earlier but good usage of pathlib instead of os.path and usage of PIL instead of directly opening the file

Reply:
:)